test/integration/com/unifina/service/PermissionServiceIntegrationSpec.groovy

```diff
@@ -39,24 +39,28 @@ class PermissionServiceIntegrationSpec extends IntegrationSpec {
 			username: "me-permission-service-integration-spec@streamr.com",
 			name: "me",
 			password: "foo",
+			lastLogin: new Date(0),
```

Comment:
Can remove `lastLogin` if you set the default field value in `SecUser.groovy` (also elsewhere in this file)